### ISSUE TITLE
User profile and changing your password should not be modal?

### ISSUE THREAD

**Issue by SCdF** [medic/medic-webapp#4098]
To be it looks quite silly, especially on mobile?

![image](https://user-images.githubusercontent.com/583851/34057379-9ee1627e-e1ce-11e7-8f95-397fe8eed65f.png)
![image](https://user-images.githubusercontent.com/583851/34057384-a52330f4-e1ce-11e7-8f76-d306a8f9258f.png)

IMO we should make it a first class page.

Thoughts @garethbowen / @sglangevin / @amandacilek ?

**Comment by garethbowen** [comment]
Definitely.

**Comment by amandacilek** [comment]
Yeah I agree, makes sense to be a real page.

**Comment by isis-meng** [comment]
Hi, I would like to pick this up if still available!

**Comment by garethbowen** [comment]
@isis-meng Yes definitely, thanks! Please let me know if you need any help getting started.

@n-orlowski Can you please weigh in with any design input for this change?

**Comment by n-orlowski** [comment]
<img width="496" alt="Screen Shot 2023-04-18 at 1 40 24 PM" src="https://user-images.githubusercontent.com/59581790/232899820-d2c10720-0cea-49aa-a2ec-7b0b3020ca24.png">

Yup! Here's an example of one of the few pages in the hamburger nav that isn't a modal. To keep it consistent we should keep the top tab bar visible (with nothing highlighted), with the contents of the current modal below as a full page. Thanks!

**Comment by isis-meng** [comment]
Hi Nicole, thanks so much for this example! I noticed that adding and editing users is also a modal rather than a page in the admin App Management view -- would this be better as a page as well, or is this issue specific to the mobile view?

**Comment by garethbowen** [comment]
@isis-meng Good spotting! I think we should handle both as separate issues as two small changes are better than one large one. There probably won't be much code shared because the app management app is still on the old AngularJS framework but the CHW app is up to date with Angular.

**Comment by isis-meng** [comment]
@garethbowen Makes sense! Do you anticipate opening a new issue for the app management app? I've already made some changes to the interface to address the issue, although I am still working on updating tests. No worries if not!

**Comment by garethbowen** [comment]
This actually may be more than just a cosmetic improvement - if this were a "real" page then users could be send a URL that loads the password change form which would make it easier to prompt users to update their password.
